File: pet-clinic-data/src/main/java/com/sachet/sfgpetclinic/services/VetService.java
package com.sachet.sfgpetclinic.services;

import com.sachet.sfgpetclinic.model.Vet;

import java.util.Set;

public interface VetService {

    Vet findById(Long id);

    Vet save(Vet vet);

    Set<Vet> findAll();

    void delete(Vet vet);

    void deleteById(Long id);

}

File: src/main/kotlin/com/sachet/sfgpetclinic/service/VetService.kt
package com.sachet.sfgpetclinic.service

import com.sachet.sfgpetclinic.model.Vet

interface VetService {
    fun findById(id: Long): Vet
    fun save(vet: Vet): Vet
    fun findAll(): Set<Vet>
}